js/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
If we don't have a clear path to using this file, or the example files under `support`, I'd remove them. If we're going to be using them approximately as soon as we start writing real tests, though, then I'd say it's fine to leave them in this PR.

Reply:
I'm not sure if we're going to use any fixtures since these will be running on actual environments (in other words, no simulated data). I think we might be using `support` files to load in environment variables though.